app/controllers/taxons_controller.rb

```diff
@@ -48,6 +48,18 @@ def update
     redirect_to taxons_path
   end
 
+  def destroy
+    resp = Services.publishing_api.unpublish(params[:id], type: "gone")
```

Comment:
Rather we didn't have the abbreviated variable name here. Reckon you can you just be rid of it by renaming it to `response_code` and moving the `.code` call below to this line.